tests/kafkatest/tests/client/client_compatibility_produce_consume_test.py

```diff
@@ -48,37 +46,13 @@ def __init__(self, test_context):
         self.messages_per_producer = 1000
         self.num_consumers = 1
 
-    def setUp(self):
-        if self.zk:
-            self.zk.start()
-
     def min_cluster_size(self):
         # Override this since we're adding services outside of the constructor
         return super(ClientCompatibilityProduceConsumeTest, self).min_cluster_size() + self.num_producers + self.num_consumers
 
     @cluster(num_nodes=9)
     @matrix(broker_version=[str(DEV_BRANCH)], metadata_quorum=quorum.all_non_upgrade)
```

Comment:
same comment here. Don't we want to add brroker versions going back to 3.4 here? That was the first GA version of KRaft.